scripts/install

```diff
@@ -57,7 +57,7 @@ then
     sudo mkdir -p $final_path
 fi
 wget https://github.com/asciimoo/searx/archive/v0.10.0.tar.gz
-tar xvf v0.10.0.tar.gz && mv searx-0.10.0 ../sources/searx
+tar xvf  v0.10.0.tar.gz && mv searx-0.10.0 searx && mv searx ../sources
```

Comment:
Ok I understand now what's wrong. searx_ynh/sources has the same content of searx repo + requirements-ynh.txt. My install script was working but was creating a searx inside searx source code…